content/artifacts/highlight-active-artifact-planets/plugin.js

```diff
@@ -0,0 +1,173 @@
+// Display the planet with active artifact, specially for cannon.
+
+import { 
+  BiomeNames
+} from "https://cdn.skypack.dev/@darkforest_eth/types";
+
+import {
+  coords
+} from 'https://plugins.zkga.me/utils/utils.js';
+
+import {
+  ArtifactType,
+  ArtifactTypeNames
+} from "https://cdn.skypack.dev/@darkforest_eth/types";
+
+const ARTIFACT_TYPES = Object.values(ArtifactType).filter((type) => type !== ArtifactType.Unknown).map((type) => ({ value: type, text: ArtifactTypeNames[type] }));
+
+
+function getPlayerColor(ethAddress) {
+  return df.getProcgenUtils().getPlayerColor(ethAddress);
+}
+
+
+function drawArc(ctx, planet, color, artifactTypes){
+  if(!planet) {
+    return false;
+  }
+
+  let artifact = df.getActiveArtifact(planet);
+  if (!artifact) {
+    return false;
+  }
+
+  if (artifactTypes.indexOf(artifact.artifactType ) == -1)  {
+    return false;
+  }
+
+  let percent = 1;
+  if (artifact.artifactType == 7) {
+    let timePass = Math.floor(Date.now() / 1000 - artifact.lastActivated);
+    percent = timePass / ( 4 * 60 * 60);
```

Comment:
i put this link here
https://github.com/darkforest-eth/client/blob/00492e06b8acf378e7dacc1c02b8ede61481bba3/src/Frontend/Panes/ArtifactDetailsPane.tsx#L193
idk maybe in the future contractConstants.PHOTOID_ACTIVATION_DELAY can be used

Reply:
I just didn't find the "contractConstants.PHOTOID_ACTIVATION_DELAY", thanks